aptos-move/aptos-vm-types/src/resource_group_adapter.rs

```diff
@@ -60,16 +70,16 @@ pub struct ResourceGroupAdapter<'r> {
 
 impl<'r> ResourceGroupAdapter<'r> {
     pub fn new(
-        _maybe_resource_group_view: Option<&'r dyn ResourceGroupView>,
+        maybe_resource_group_view: Option<&'r dyn ResourceGroupView>,
         resource_view: &'r dyn TResourceView<Key = StateKey, Layout = MoveTypeLayout>,
         gas_feature_version: u64,
     ) -> Self {
         let group_size_kind = GroupSizeKind::from_gas_feature_version(gas_feature_version);
 
         Self {
-            // TODO: when ResourceGroupView is implemented by block executor, provide
-            // (group_size_kind == GroupSizeKind::AsSum).then(|| maybe_resource_group_view)
-            maybe_resource_group_view: None,
+            maybe_resource_group_view: (group_size_kind == GroupSizeKind::AsSum)
```

Comment:
We need to feature gate here
(and also whenever we connect to capable, shouldn't set if not capable)

Reply:
@igor-aptos